services/web/server/tests/unit/with_dbs/03/test_users.py

```diff
@@ -789,3 +791,56 @@ async def test_list_permissions_with_no_group_defined_returns_default_false_for_
 
     assert override_services_specifications.name == "override_services_specifications"
     assert override_services_specifications.allowed is False
+
+
+@pytest.mark.parametrize("user_role", [UserRole.USER])
+async def test_mark_account_for_deletion(
+    client: TestClient, logged_user: UserInfoDict, mocker: MockerFixture
+):
+    mock = mocker.patch(
+        "simcore_service_webserver.email._core._do_send_mail",
+        spec=True,
+    )
+
+    # is logged in
+    response = await client.get("/v0/me")
+    await assert_status(response, web.HTTPOk)
+
+    # failed check to delete account
+    response = await client.post(
+        "/v0/me:mark-deleted",
+        json={
+            "email": "WrongEmail@email.com",
+            "password": "foo",
+        },
+    )
+    await assert_status(response, web.HTTPConflict)
+
+    # success to request deletion of account
+    response = await client.post(
+        "/v0/me:mark-deleted",
+        json={
+            "email": logged_user["email"],
+            "password": logged_user["raw_password"],
+        },
+    )
+    await assert_status(response, web.HTTPOk)
```

Comment:
you do not test the response contents although it's a 200. why?
should it not be a 204 if the response is useless and/or there is no response? or otherwise shouldn't you check the contents?